model_analyzer/result/result_table_manager.py

```diff
@@ -35,6 +35,7 @@ class ResultTableManager:
         'batch_size': 'Batch',
         'concurrency': 'Concurrency',
         'model_config_path': 'Model Config Path',
+        'ensemble_subconfig_path': 'Ensemble Subconfig Path',
```

Comment:
Is this header used? I don't see it in the images

Reply:
Good catch. This is a remnant from when I was going to make the subconfigs a separate column.